src/broadcast/operations.js

```diff
@@ -487,6 +487,107 @@ module.exports = [
       "extensions"
     ]
   },
```

Comment:
What about additions to existing ops (e.g. `comment_options`) in `src/auth/serializer/src/operations.js`? If they cannot be added until HF, please document these cases in an issue.

Reply:
The `extensions` field is defined as a set of static variant. Since there is no new field, there is nothing to add here.